apis/Google.Cloud.Logging.Console/Google.Cloud.Logging.Console/GoogleCloudConsoleFormatterOptions.cs

```diff
@@ -20,8 +20,7 @@ namespace Google.Cloud.Logging.Console;
 /// Options for <see cref="GoogleCloudConsoleFormatter"/>.
 /// </summary>
 /// <remarks>
-/// This class does not currently introduce any new formatter options,
-/// but exists to allow further expansion without disruptive API changes.
+/// This class exists to allow further expansion without disruptive API changes.
```

Comment:
I think this is not necessary anymore. When options are empty it makes sense to explain why, but not we have options, so we can just remove the whole remarks section.